Makefile

```diff
@@ -111,7 +111,7 @@ lint-lua: FORCE
 lint-sh: FORCE
 	@scripts/lint-sh.sh
 
-GLUON_DEFAULT_PACKAGES := hostapd-mini
+GLUON_DEFAULT_PACKAGES :=
```

Comment:
The variable `GLUON_DEFAULT_PACKAGES` and the `merge_lists` using it can be removed as well.